### PR TITLE
python: restore subliminal in python3 set

### PR DESCRIPTION
**Pull request by dguibert**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 122853` at 6ee84d8d run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.subliminal</li>
    <li>python39Packages.subliminal</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/subliminal/default.nix:51:3:
```
   |
51 |   meta = with lib; {
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md">python-include-tests</a>

Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/subliminal/default.nix:53:0:
```
   |
53 |     description = "Python library to search and download subtitles";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/subliminal/default.nix [hunk 1]
```diff
@@ -16,7 +16,7 @@
 , appdirs
 , rarfile
 , pytz
-, futures
+, futures ? null
```
Comment (davidak) [inline pkgs/development/python-modules/subliminal/default.nix:19]: what does this?

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -7890,6 +7890,8 @@ in {
 
   subdownloader = callPackage ../development/python-modules/subdownloader { };
 
+  subliminal = callPackage ../development/python-modules/subliminal { };
```
Comment (davidak) [inline pkgs/top-level/python-packages.nix:7893]: it seem to be there in master

https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/python-packages.nix#L8435

File: pkgs/development/python-modules/subliminal/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/subliminal/default.nix:19]: futures is only available in python2 IIRC.

**Pull request closed without merging**